teuthology/misc.py

```diff
@@ -550,27 +550,31 @@ def copy_file(from_remote, from_path, to_remote, to_path=None):
     ])
 
 
-def move_file(remote, from_path, to_path, sudo=False):
+def move_file(remote, from_path, to_path, sudo=False, preserve_perms=True):
```

Comment:
I guess that's the part that rankles a bit; it's changing the intent of the original definition, however misguided, while leaving the default behavior, but not going the step further to add the capability of choosing the actual permission.  But I can see that it's certainly a matter of opinion which way is more in keeping with expectations.  I withdraw the suggestion.